### ISSUE TITLE
Questions about task tuning in medical domain

### ISSUE THREAD

**Issue by yiyiwwang** [OptimalScale/LMFlow#756]
Dear authors,  I have some questions about the Task Tuning in your paper LMFlow:
1. Do you use the training set of PubMedQA and MedMCQA to get the LLaMA-7/13/30B-LoRA models? And then test on the validation/test set of PubMedQA and MedMCQA (in-domain) and MedQA-USMLE (out-of-domain), right? I‘m curious about if there is base LLaMA 30B model? 
2. The task tuning is continuous pretraining, right? The continuous pretraning is occured on the LoRA, not full parameters fine-tuning, is my understanding right? If so, will the differece be large between LoRA tuning and Full tuning in the continuous pretraining?
3. In your tuned medical models, do you find any forgetting problems about ability loss in general tasks? In the continuous pretraining, do you think it is helpful to add general data to allieviate this problem? Can you give me some suggestions about how to add the general data?
Thank you very much.

**Comment by research4pan** [comment]
Thanks for your interest in LMFlow! @2003pro Can you please take a look?

**Comment by yiyiwwang** [comment]
Thanks very much for your clear explanation, and your suggestions are helful for me. 
BTW, I want to confirm that does your mention "like our 1B tokens to get it to work" refer to your LMFlow Dataset (http://lmflow.org:5000/lmflow_data.tar.gz) ?

**Comment by research4pan** [comment]
I just asked @2003pro, we think it is more like some general medical dataset, like pubmed central in pile (https://github.com/EleutherAI/pile-pubmedcentral). Hope this information can be helpful 😄

**Comment by yiyiwwang** [comment]
> I just asked @2003pro, we think it is more like some general medical dataset, like pubmed central in pile (https://github.com/EleutherAI/pile-pubmedcentral). Hope this information can be helpful 😄

Thank you very much. I will learn it.
